packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -613,6 +613,24 @@ const TagSubforumPage2 = ({classes}: {
               />
             )
           },
+          tagSubforumStickyComments: {
+            fragmentName: "StickySubforumCommentFragment",
+            render: (comment: CommentWithRepliesFragment) => (
+              <CommentWithReplies
+                key={comment._id}
+                comment={{...comment, isPinnedOnProfile: true}}
+                commentNodeProps={{
+                  ...commentNodeProps,
+                  showPinnedOnProfile: true,
+                  treeOptions: {
+                    ...commentNodeProps.treeOptions,
+                    showPostTitle: true,
+                  },
+                }}
+                initialMaxChildren={5}
```

Comment:
I think we should go lower than this (3) so people's replies don't take up too much space, and also make sure replies start collapsed if they are not already